game/scripts/vscripts/internal/util.lua

```diff
@@ -6,14 +15,88 @@ function split(s, delimiter)
     return result;
 end
 
+function regexsplit(s, delimiter)
+    result = {};
+    for match in s:gmatch("([^"..delimiter.."]+)") do
+        table.insert(result, match);
+    end
+    return result;
+end
+
+function TracesFromFilename (filename)
+  local traces = {}
+  local i = 1
+
+  local parts = regexsplit(filename, '%s/\\')
```

Comment:
I'm splitting on `%s`, `/`, and `\` here. The regex split method takes in the string and puts it inside a `[^ delim ]` segment, so here it'll grab whitespace or either platform's slash.